shared/js/background/components/dashboard-messaging.js

```diff
@@ -69,7 +69,36 @@ export default class DashboardMessaging {
         if (!tab) {
             return;
         }
-        const pageParams = (await browser.tabs.sendMessage(tab.id, { getBreakagePageParams: true })) || {};
+
+        // Try to get breakage data from content-scope-scripts, with fallback to legacy
+        let pageParams = {};
+        try {
+            // Send message to content-scope-scripts via content script
+            // Note: must use messageType format for content-scope-scripts messaging
+            await browser.tabs.sendMessage(tab.id, {
+                messageType: 'getBreakageReportValues',
+            });
+
+            // Wait briefly for the breakageReportResult handler to receive and store data
+            await new Promise((resolve) => setTimeout(resolve, 500));
+        } catch (e) {
+            // Content-scope-scripts not available or timed out, will fallback to legacy
+            console.log('Content-scope-scripts breakage reporting not available, using legacy approach');
+        }
+
+        // Get page params (legacy approach fallback)
+        try {
+            pageParams = (await browser.tabs.sendMessage(tab.id, { getBreakagePageParams: true })) || {};
```

Comment:
Is this needed, seems like usual GPT 'fallback' stuff?

Reply:
Turns out the legacy script does more than jsPerformance. It's also detects page reloads. I'd rather just leave it for now, and make a followup to migrate over to content scopes